openshift/release/generate-release.sh

```diff
@@ -13,4 +13,4 @@ else
     tag=$release
 fi
 
-resolve_resources config/core/ "$output_file" "$image_prefix" "$tag"
+resolve_resources "config/core/ config/hpa-autoscaling/" "$output_file" "$image_prefix" "$tag"
```

Comment:
hap deployments are necessary for https://github.com/knative/serving/blob/master/test/ha/autoscalerhpa_test.go.